### DISCUSSION TITLE
Icon request: fa-yoga

### DISCUSSION THREAD

**Discussion by jweblog** [FortAwesome/Font-Awesome#21141] [Icon Requests]
fa-yoga  
~fa-om
ॐ~

**Comment by G-Driver** [comment]
+1 why not

**Comment by mperino** [comment]
Id like to humbly +1 a requested an Icon for Yoga. I think warrior 1 pose would be generic enough and have a lot of use.

eg:http://www.popsugar.com/fitness/photo-gallery/33065422/image/33065563/Warrior-1

Much thanks, and I love the Icons youve already done. The lock, facebook, email, and shopping cart are the ones I use most so far. I started using them thanks to Pixelarity.

Thank you for the AWESOME work so far, and for being open to requests for new icons.

**Comment by elleeye** [comment]
+1

**Comment by sumitgulati100** [comment]
+1

**Comment by nirmalavidya** [comment]
+1

**Comment by WebVantage** [comment]
+1

**Comment by stefanpl** [comment]
+1

**Comment by karthikandru** [comment]
+1

**Comment by jsjoeio** [comment]
+1

**Comment by rawthriver** [comment]
+1

**Comment by baitasite** [comment]
+1

**Comment by andreescocard** [comment]
+1

**Comment by mrmikemcguire** [comment]
+1

**Comment by russellmann** [comment]
+1

**Comment by emmahunter** [comment]
+1

**Comment by earid** [comment]
+1

**Comment by NiveouSs** [comment]
+1

**Comment by laurensvo92** [comment]
+1

**Comment by MahsanNourani** [comment]
+1

**Comment by amkuchta** [comment]
I would go so far as to make this an entire icon pack, using the poses in [this link](https://greatist.com/move/common-yoga-poses#intermediate) as inspiration.

**Comment by stefanpl** [comment]
It's really comforting being subscribed to this issue. Every now and then I get an email, and I take it as a sign for the world still being okay, in its own weird ways.

**Comment by tagliala** [comment]
The symbol `om` is a duplicate of #3522, so please +1 that request if you are interested in the symbol

Let's leave this open for `yoga`, as a yoga position. Classifying under sport category, even if it is not correct

**Comment by amkuchta** [comment]
Would this be the appropriate place to suggest that `yoga` become its own separate category? Or would that be another issue that needs to be submitted (referencing back to this one)?

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🧘 [U+1F9D8](https://emojipedia.org/emoji/%F0%9F%A7%98/) Person in Lotus Position
Also known as `Meditation` and `Yoga`.

**Comment by armandsLa** [comment]
+1

Please consided this as a separate category, dedicated to wellbeing/mindfulness.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
> _Related Unicode Emoji_ [#222](https://github.com/FortAwesome/Font-Awesome/issues/222) 🧘 [U+1F9D8](https://emojipedia.org/emoji/%F0%9F%A7%98/) Person in Lotus Position Also known as `Meditation` and `Yoga`.

Would an icon like this cover this request? Or is the preference for a more active pose?

**Comment by amkuchta** [comment]
I'd actually make the suggestion that yoga could be its own category, if you really wanted - with so many well-known poses (tree, warrior, lotus, etc.), there are a ton of options.
